include/zenoh-pico/collections/bytes.h

```diff
@@ -34,9 +34,11 @@ typedef struct {
 } _z_bytes_t;
 
 _z_bytes_t _z_bytes_empty(void);
+inline static _Bool _z_bytes_check(_z_bytes_t value) { return value.start != NULL; }
 int8_t _z_bytes_init(_z_bytes_t *bs, size_t capacity);
 _z_bytes_t _z_bytes_make(size_t capacity);
 _z_bytes_t _z_bytes_wrap(const uint8_t *bs, size_t len);
+_z_bytes_t _z_bytes_steal(_z_bytes_t *b);
```

Comment:
Wasn't '_z_bytes_move` already implementing the same behavior?

Reply:
it was indeed, didn't spot it because the behaviour differs to other moves. I think `_z_bytes_move` will be renamed to the steal version, to avoid being confused for the move convention